File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ConditionalStatementsMoreExercises/Pets.java
package ConditionalStatements.ConditionalStatementsMoreExercises;

import java.util.Scanner;

public class Pets {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int days = Integer.parseInt(scan.nextLine());
        int kgFoodLefted = Integer.parseInt(scan.nextLine());
        double kgDogDayfood = Double.parseDouble(scan.nextLine());
        double kgCatDayFood = Double.parseDouble(scan.nextLine());
        double kgTurtleDayFood = Double.parseDouble(scan.nextLine()) / 1000;

        double totalKgFoodNeeded = days * kgDogDayfood + days * kgCatDayFood + days * kgTurtleDayFood;

        if (kgFoodLefted >= totalKgFoodNeeded) {
            System.out.printf("%.0f kilos of food left.", Math.floor(kgFoodLefted - totalKgFoodNeeded));
        } else {
            System.out.printf("%.0f more kilos of food are needed.", Math.ceil(totalKgFoodNeeded - kgFoodLefted));
        }
    }
}

File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/pets.js
function pets(input) {

    let days = Number(input[0])
    let foodLeftKg = Number(input[1])
    let dogDayfoodKgPerDay = Number(input[2])
    let catDayfoodKgPerDay = Number(input[3])
    let turtleDayfoodGramsPerDay = Number(input[4])

    let totalFood = days * (dogDayfoodKgPerDay + catDayfoodKgPerDay + (turtleDayfoodGramsPerDay / 1000))

    if (totalFood > foodLeftKg) {
        console.log(`${Math.ceil(totalFood - foodLeftKg)} more kilos of food are needed.`)
    }
    else { console.log(`${Math.floor(foodLeftKg - totalFood)} kilos of food left.`) }

}
pets(["2", "10", "1", "1", "1200"])